hub/templates/hub/area.html

```diff
@@ -159,5 +165,6 @@ <h2 class="mt-5 mb-3 text-primary">Movement</h2>
 
     </div>
 </div>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.min.js" integrity="sha384-cn7l7gDp0eyniUwwAZgrzD06kc/tftFf19TOAs2zVinnD/C7E91j9yyk5//jjpt/" crossorigin="anonymous"></script>
```

Comment:
What is this for?

Reply:
Oops, this is a hold-over from previously messing around with displaying the APPGs. It's been removed, now